documentation/operators/src/nodes/maintenance.md

```diff
@@ -205,33 +205,79 @@ This lets your operating system know it's ok to reload the service configuration
 
 ## Moving a node
 
-In case of a need to move a node from one machine to another and avoiding to lose the delegation, here are few steps how to do it.
-
-The following examples transfers a Mix Node (in case of other nodes, change the `mixnodes` in the command for the `<NODE>` of your desire.
-
-* Pause your node process.
+In case of a need to move a Nym Node from one machine to another and avoiding to lose the delegation, here are few steps how to do it.
 
 Assuming both machines are remote VPS.
 
-* Make sure your `~/.ssh/<YOUR_KEY>.pub` is in both of the machines `~/.ssh/authorized_keys` file
+* Make sure your `~/.ssh/<YOUR_KEY>.pub` is in both of the servers `~/.ssh/authorized_keys` file
 * Create a `nym-nodes` folder in the target VPS. SSH in from your terminal and run:
 
 ```sh
 # in case none of the nym configs was created previously
 mkdir ~/.nym
 
-#in case no nym Mix Node was initialized previously
+#in case no nym Nym Node was initialized previously
 mkdir ~/.nym/nym-nodes
 ```
-* Move the node data (keys) and config file to the new machine by opening your **local terminal** (as that one's ssh key is authorized in both of the machines) and running:
+* Move the node data (keys) and config file to the new machine by opening your **local terminal** (as that one's ssh key is authorized in both of the VPS) and running:
 ```sh
 scp -r -3 <SOURCE_USER_NAME>@<SOURCE_HOST_ADDRESS>:~/.nym/nym-nodes <TARGET_USER_NAME>@<TARGET_HOST_ADDRESS>:~/.nym/nym-nodes/
 ```
-* Re-initialise (`run` command) the node to generate a config with the new listening address.
+
+**On new/target machine**
+
+* Edit `~/.nym/nym-nodes/<ID>/config/config.toml` config with the new listening address IP.
+* Setup the [systemd](#systemd) automation, reload the daemon and run the service, or just simply run the node if you don't use automation
 * Change the node smart contract info via the wallet interface. Otherwise the keys will point to the old IP address in the smart contract, and the node will not be able to be connected, and it will fail up-time checks.
-* Re-run the node from the new location.
 
+## Rename node local ID
+
+Local node ID (not the identity key) is a name chosen by operators which defines where the nodes configuration data will be stored, where the ID determines the path to `~/.nym/nym-nodes/<ID>/`. This ID is never shared on the network.
+
+Since the migration to [`nym-node`](nym-node.md), the use of ID as a flag `--ID <ID>` is no longer required when starting a new node. Nodes without an ID specified will be asigned an ID `default-nym-node`. The management of nodes is simplified and it makes it especially easier for operators managing multiple nodes through ansible and other automation scripts, as everything is stored at `~/.nym/nym-nodes/default-nym-node`.
```

Comment:
since migrating to [nym-node]... specifying an ID with --ID <ID> when starting a new node is no longer necessary. nodes without a specified ID will be assigned the default ID default-nym-node. this streamlines node management, particularly for operators handling multiple nodes via ansible and other automation scripts, as all data is stored at ~/.nym/nym-nodes/default-nym-node

Reply:
done